cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaExporterTest.java

```diff
@@ -268,7 +268,7 @@ public void testContainerTemplateEnvVarsWithKeyConflict() {
                 .build();
 
         Kafka resource = ResourceUtils.createKafkaCluster(namespace, cluster, replicas, image,
-                healthDelay, healthTimeout, metricsCm, kafkaConfig, zooConfig,
+                healthDelay, healthTimeout, metricsCm,  kafkaConfig, zooConfig,
```

Comment:
why this additional space? This is a test file that should not be impacted by the JMX options implementation.